lib/windows/app/index.js

```diff
@@ -61,6 +61,18 @@ Module._load = function load(path) { // eslint-disable-line no-underscore-dangle
             return ReactDOM;
         case 'react-redux':
             return ReactRedux;
+        case 'nrfconnect/logger':
+            return api.logger;
+        case 'nrfconnect/pc-ble-driver-js':
+            return api.bleDriver;
+        case 'nrfconnect/serialport':
+            return api.SerialPort;
+        case 'nrfconnect/programming':
+            return api.programming;
+        case 'nrfconnect/electron':
+            return api.electron;
+        case 'nrfconnect/core':
+            return api.core;
```

Comment:
What do you think about importing the logger from `nrfconnect/core`?

```
import { logger } from 'nrfconnect/core';
```

Logically, I think it makes sense to keep everything closely related to core in `nrfconnect/core`. External libraries that are not core-specific can be made available on `nrfconnect/<library-name>`.

When the high-level nrfjprog API is ready, I suggest replacing `nrfconnect/programming` with `nrfconnect/pc-nrfjprog-js`. This way, it is clear to the developer that he/she is dealing with an external library, and we can simply refer to the API doc for that library.